bindings/src/main/java/com/symphony/api/bindings/jersey/JerseyApiBuilder.java

```diff
@@ -54,7 +54,7 @@ public JerseyApiBuilder(String url, KeyManager[] keyManagers) {
 		return buildProxy(c, wt);
 	}
 	
-	protected WebTarget newWebTarget(String url) {
+	public WebTarget newWebTarget(String url) {
```

Comment:
Any specific reason access specifier is changed?

Reply:
Not a good reason, this is breaking encapsulation and I'm going to change it back.